rclpy/src/rclpy/_rclpy.c

```diff
@@ -834,7 +830,7 @@ rclpy_destroy_node_entity(PyObject * Py_UNUSED(self), PyObject * args)
   }
 
   rcl_node_t * node = (rcl_node_t *)PyCapsule_GetPointer(pynode, NULL);
-  rcl_ret_t ret;
+  rcl_ret_t ret = RCL_RET_ERROR;
```

Comment:
This initial value is never used, why did you add it throughout the file? My static linter complains about this ("value assigned but never used").